prisma/seed.ts

```diff
@@ -3,71 +3,78 @@ import { PrismaClient } from "@prisma/client";
 const prisma = new PrismaClient();
 
 async function main() {
-  const deck = await prisma.deck.create({
+  const user = await prisma.user.create({
```

Comment:
This diff is really ugly, but it's really just a change to add a user with a deck array which in turn has a card array. One more layer of confusion!